justfile

```diff
@@ -58,7 +58,13 @@ lint-manual *args="": && resync
 
 [group('test')]
 test *args="-v --durations=5 --maxfail=10": && resync
-    uv run --extra test pytest {{args}}
+    uv run --extra test python -m pytest {{args}}
+
+[group('test')]
+test-bson *args="-v --durations=5 --maxfail=10": && resync
```

Comment:
I think this is confusing, there might be other place's we'd use numpy optionally, perhaps `test-numpy`?

Reply:
I say go for it